openhtf/conf.py

```diff
@@ -238,6 +239,7 @@ def __init__(self, logger, lock, parser, **kwargs):
     self._modules = kwargs
     self._declarations = {}
     self.ARG_PARSER = parser
+    self.__name__ = 'Configuration'
```

Comment:
I believe you can actually do type(self).__name__ here, as the `Configuration` type still exists when `__init__` is called (just not when anything else is called).